index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta name="Author" content="Paul A. Gureghian.">
+    <meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width" initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Apple</title>
+    <link href="styles.css" type="text/css" rel="stylesheet" media="screen"> 
+</head>
+
+<body>
+  <div class="container">
+	<header class="hero">
+	  <nav class="main-nav">
+		  <ul>
+			<li id="apple"><a href="/web/20140301004610/http://www.apple.com/"></a></li>                
+			<li id="store"><a href="https://web.archive.org/web/20140301004610/http://store.apple.com/"></a></li>
+			<li id="mac"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/mac/"></a></li>
+			<li id="ipod"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipod/"></a></li>
+			<li id="iphone"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/iphone/"></a></li>
+			<li id="ipad"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipad/"></a></li>
+			<li id="itunes"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/itunes/"></a></li>
+			<li id="support"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/support/"></a></li>
+			<li id="search"></li>
```

Comment:
The search bar should be an `<input type="text" class="search">` then style the class search in the CSS file,  and not a CSS image